src/server/req_runjob.c

```diff
@@ -1084,7 +1054,10 @@ svr_strtjob2(job *pjob, struct batch_request *preq)
 			pjob->ji_qs.ji_jobid,
 			"Unable to Run Job, send to Mom failed");
 
-		if (pjob->ji_qs.ji_substate == JOB_SUBSTATE_PROVISION)
+		mark_node_down(pjob->ji_qs.ji_destin, "could not send job to mom");
```

Comment:
can send_job not fail due to multiple reasons? How will you know that under what kind of failure you want to mark the node down?

Reply:
Nice catch, fixed in commit with tag "rc5"